lib/string.gi

```diff
@@ -1016,29 +1021,29 @@ InstallGlobalFunction(PrintCSV,function(arg)
     end);
   fi;
 
-  PrintTo(file);
+  PrintTo(stream);
 
   if ValueOption("noheader")<>true then
     printEntry(rf[1]);
     for j in [2..Length(rf)] do
-      AppendTo(file,",");
+      AppendTo(stream,",");
       printEntry(ReplacedString(rf[j],"_"," "));
     od;
-    AppendTo(file,"\n");
+    AppendTo(stream,"\n");
   fi;
 
   for  i in l do
     for j in [1..Length(rf)] do
       if j>1 then
-	AppendTo(file,",");
+        AppendTo(stream,",");
       fi;
       if IsBound(i.(rf[j])) then
         printEntry(i.(rf[j]));
       fi;
     od;
-    AppendTo(file,"\n");
+    AppendTo(stream,"\n");
   od;
-  SizeScreen(sz);
+  SetPrintFormattingStatus(stream,true);
```

Comment:
It would be better to restore the previous value, instead of blindly reseting to `true` here. I.e. use `PrintFormattingStatus` at the start of the function to store the original value, then restore here to that.

Reply:
Yes, that sounds like a good idea. I'll change that and add a test as you described. Thanks for the feedback!

Reply:
Good! Could you add that example as a test into a .tst file? Feel free to ask if you need to know details.

Reply:
I did it.